docs/conf.py

```diff
@@ -73,7 +73,7 @@
 # (source start file, target name, title,
 #  author, documentclass [howto, manual, or own class]).
 latex_documents = [
-    (master_doc, 'flakeheaven.tex', 'FlakeHeaven Documentation',
+    (main_doc, 'flakeheaven.tex', 'FlakeHeaven Documentation',
```

Comment:
```suggestion
    (root_doc, 'flakeheaven.tex', 'FlakeHeaven Documentation',
```

same, https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-master_doc